src/coreclr/vm/exceptionhandling.h

```diff
@@ -39,6 +44,16 @@ enum EHFuncletType
     Catch = 0x0004,
 };
 
+struct ExInfo;
+typedef DPTR(ExInfo) PTR_ExInfo;
+
+enum class InlinedCallFrameMarker
+{
+    ExceptionHandlingHelper = 2,
```

Comment:
If the 1 bit is important we should document that here or point to where it is defined in another enum.

Reply:
Preexisting code is using the bit 0 for unrelated purposes, it is described in comment on the m_Datum field. I'll add a note here.